kiwi/container/oci.py

```diff
@@ -117,7 +128,7 @@ def create(self, filename, base_image):
             image_tar = ArchiveTar(base_image)
             image_tar.extract(self.oci.container_dir)
 
-        self.oci.init_layout(base_image)
+        self.oci.init_layout(True if base_image else False)
```

Comment:
base_image is None if not set so that evaluates the same as False in the way init_layout is coded. If set it's a string which evaluates as True in the way init_layout is coded. But I agree this change makes it more clear and bool correct

Reply:
just a side note

Reply:
Couldn't you do `bool(base_image)`?

Reply:
@cyphar I haven't thought about that. Yes that's probably nicer, just need to make some tests on python2 and 3 to make sure it does not evaluate different.